### PR TITLE
Scrub access_token from breadcrumb data sent to sentry

### PR DESCRIPTION
**Pull request by ahie**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -14,6 +14,12 @@ const { REACT_APP_SENTRY_DSN, REACT_APP_SENTRY_ENVIRONMENT } = process.env;
 Sentry.init({
   dsn: REACT_APP_SENTRY_DSN,
   environment: REACT_APP_SENTRY_ENVIRONMENT,
+  beforeBreadcrumb(breadcrumb) {
+    if (JSON.stringify(breadcrumb.data).includes('access_token')) {
+      delete breadcrumb.data;
```
Comment (Nizar-Rahme) [inline src/index.tsx:19]: Wondering if we should omit the whole `data` property or getting rid of only `access_token` would do the trick.

Comment (TatuArvela) [reply src/index.tsx:19]: Any update on this? Should we omit more fields or merge this as it is?

**Review by TatuArvela — APPROVED** [review]
Let's get this merged already 👍

**Comment by Nizar-Rahme** [comment]
👍

**Pull request merged**